.circleci/config.yml

```diff
@@ -73,8 +73,10 @@ workflows:
           mapping: |
             packages/web/.* run-web-workflow true
             packages/mobile/.* run-mobile-workflow true
+            packages/stems/.* run-stems-workflow true
             .circleci/.* run-web-workflow true
             .circleci/.* run-mobile-workflow true
+            .circleci/.* run-stems-workflow true
```

Comment:
This isn't expressly necessary bc we don't have any CI that runs on stems conditionally, but good to have in case we ever add it